content/docs/3.0/contribute/get-started.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Contribute
```

Comment:
We can't have pages with the same name in Hugo menus :( So one of the pages named `Contribute` needs to change.

Reply:
ACtually, this entire page seems to be a duplicate. Please delete one and rename the remaining one to `content/docs/3.0/contribute/introduction.md` to keep things consistent.

Reply:
Sorry - this was a careless duplicate happened when I renamed the file